app/controllers/editorial/news_controller.rb

```diff
@@ -6,7 +6,7 @@ class NewsController < EditorialController
 
     def index
       # TODO: return news editorial index
-      bustable_fresh_when(@sections)
+      with_caching(@sections)
```

Comment:
That particular invocation will only change the etag based on `@section.updated_at`.